CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Mongo Spark Connector Changelog
 
+## 3.1.0
```

Comment:
nit: 3.1.1? Why not to match Spark version?

Reply:
Because this is the first release of the 3.1 series and there have been no patches to the connector.